index.js

```diff
@@ -0,0 +1,11 @@
+let menuSection = document.querySelector('#mobile-menu')
+let menu = menuSection.querySelector(".icon-wrapper");
+menu.addEventListener('click', () => {
+    menuSection.classList.add("d-none");
+});
```

Comment:
- [ ] When I change the size of the screen from desktop to mobile, it automatically opens the mobile menu. However, the mobile menu should open only when the humburger button is clicked: 

https://user-images.githubusercontent.com/77038610/189679891-056f1b76-e14a-4b38-b2c0-90a4c4461519.mp4

Please make the mobile menu invisible if the humburger button was not clicked by a user. 



- [ ] Hint: To make the menu  invisible add `d-none` class to `menuSection`, then  use `toggle`  in JS to delete or add class:

```
#html
<section id="mobile-menu" class="d-none">
```

```
# JS
menu.addEventListener('click', () => {
  menuSection.classList.toggle('d-none');
});

hamburgerIcon.addEventListener('click', () => {
  menuSection.classList.toggle('d-none');
}); 
```

Reply:
Got it thanks